cloudca/resource_cloudca_vpn_test.go

```diff
@@ -0,0 +1,99 @@
+package cloudca
+
+import (
+	"fmt"
+	"testing"
+
+	cca "github.com/cloud-ca/go-cloudca"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccRemoteAccessVPNEnable(t *testing.T) {
+	/*
+		test is run in series since it uses a vpn that is changes
+		in another test
+	*/
+	// t.Parallel()
```

Comment:
can probably delete this since you have the comment explanation

Reply:
d5b60431492ddeaac993ad92a69950618377e48b